m4_sessions/1711-inning/Group_2.md

```diff
@@ -1,9 +1,19 @@
 Group Two
 
 Session Title: Description: Why should students come to this session?
+Because Turing is designed around failure and how to learn from that failure. This class seeks to teach how to more effectively learn from that struggle.
 
 Session Outcomes: What will students leave with or gain from this session?
+How to see failure as a stepping stone to future successes.
 
 Session Outline: Include the flow of the session and the time for each section.
+15 Minutes - Student introductions and why we're here
+30 Minutes - Presentation regarding struggle
+30 Minutes - Small group discussion / Practical excercise
+15 Minutes - Wrap up / Whole group share
```

Comment:
Be specific about how the group will share.  Does everyone share their closing thoughts?  Does one person share the groups ideas?  Do you want to have everyone put ideas on the whiteboard?